### ISSUE TITLE
Severity & Tag computed incorrectly

### ISSUE THREAD

**Issue by ianawilson** [jviotti/node-crowsnest#1]
You're currently calculating the severity & tag field by shifting the severity by the tag:

```
options.severity << options.tag
```

But it should be that least significant bit should be a 0, the severity is the next 2 bits, while the tag is the five most significant bits. Put another way, the binary representation is `0SSTTTTT`, where `SS` is the severity and `TTTTT` is the tag. Given that, I believe the following change would report the correct severity and tag:

```
(options.severity << 5) + options.tag
```

**Comment by jviotti** [comment]
@ianawilson Thanks for the catch. I've implemented and tested the fix, which correctly sets the severity in Crow's Nest.

**Comment by ianawilson** [comment]
Great! Glad to hear it.

**Comment by jviotti** [comment]
@ianawilson The fix was published to NPM as v0.0.2.

**Comment by ianawilson** [comment]
Awesome, thanks! :D
